src/api/service/Movies.js

```diff
@@ -0,0 +1,9 @@
+import axios from "@/api/axiosConfig.js";
```

Comment:
I actually copied this solution verbatim from Mateusz's repo that he shared with us during the meeting about API connections:
https://github.com/MateuszMichalowski/communication-with-api/tree/main/src/api

What name would you suggest in this case? I'm totally fresh with axios so I don't feel yet what kind of confusion it may create.

Reply:
Hope Mateusz won't see that I question his approach 😬 I have one concern thinking about this. If we would like to import both which one would be `axios`. Both but one is configured, and with time going, probably it will be even more. The conclusion is they are different. It is not so crucial maybe just my whims but simply `axiosClient` will do the job in my opinion 😅

Reply:
I got your point. But for the purposes of this app I think it could stay as is. I import axios from axios once, I set it up in the config file and then export it configured. All other imports will use only the configured axios and the import's path will make it clear. What do you think?

Reply:
For sure it can stay as it is 👌